packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md

```diff
@@ -0,0 +1,224 @@
+
+You need to provide a configuration guide as part of the OIN submission process. Your guide is provided to administrators through the Okta Admin Console. Okta checks your document for general adherence to the configuration guide instructions. After your integration is published to the OIN, you can make the link public or customer‐accessible.
+
+>**Note:** A configuration guide is required for SCIM and OIDC integrations. It's optional for SAML integrations, as Okta supplies its own documentation with the apps.
+
+You can create the guide in whatever format works best for you: a webpage, a Google or Word doc, or a PDF are all acceptable.
+
+### Configuration guide examples
+
+OIDC:
+
+* [Cerby](https://docs.google.com/document/d/e/2PACX-1vRiswyNyRPVYfEMEwPsbMO8Qn11BjAO-FfUsWBBit_IYe88tzQCHTmMPtmF8uPV044HmXQR13adj3LO/pub)
+* [Upwave](https://help.upwave.io/en/articles/4129778-okta-configuration-guide)
+
+SAML:
+
+* [GitHub Enterprise](https://saml-doc.okta.com/SAML_Docs/How-to-Configure-SAML-2.0-for-Github_Enterprise.html)
+* [Runscope](https://saml-doc.okta.com/SAML_Docs/How-to-Configure-SAML-2.0-for-Runscope.html)
+* [Salesforce](https://saml-doc.okta.com/SAML_Docs/How-to-Configure-SAML-2.0-in-Salesforce.html)
+* [Zoom.us](https://saml-doc.okta.com/SAML_Docs/How-to-Configure-SAML-2.0-for-Zoom.us.html)
+
+SCIM:
+
+* [Atlassian Cloud](https://confluence.atlassian.com/cloud/configure-user-provisioning-with-okta-957492956.html)
+* [Contentful](https://www.contentful.com/help/okta-user-provisioning-integration-with-scim/)
+* [Zscaler](https://help.zscaler.com/zia/saml-scim-configuration-guide-okta)
+
+### Configuration guide content
+
+The following are section suggestions for your configuration guide:
+
+* [Prerequisites](#prerequisites)
+* [Supported features](#supported-features)
+* [Procedure](#procedure)
+* [Troubleshooting tips](#troubleshooting-and-tips)
+
+#### Prerequisites
+
+In this section, specify any prerequisites required before your customer configures your integration in Okta. Examples may include enabling specific Okta features, enabling API access to your SCIM server, or adding a particular version of an integration in Okta.
+
+For example:
+
+When using SAML as the SSO mode with provisioning, you need to enable a specific account plan on the application side for silent activation.
+
+#### Supported features
+
+In this section of your guide, you want to outline what features your application supports. For example:
+
+* IdP-initiated SSO
+* SP-initiated SSO
+* SLO (Single Log Out)
+* Force Authentication
+* Create Users
+* Update User Attributes
+* Deactivate Users
+* Import Users
+* Import Groups
+* Sync Password
+* Profile Sourcing (formerly called Profile Mastering)
+
+Also include any restrictions. For example:
+
+Okta can't update user attributes for Admin users. This is an API limitation.
+
+> **Note:** You can briefly describe what each feature does. See the SCIM guides from the earlier example section.
```

Comment:
Do we have a description somewhere so that people don't need to come up with their own descriptions?